src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -1,17 +1,21 @@
 package edu.oregonstate.cs361.battleship;
 
+import com.sun.javafx.image.BytePixelSetter;
```

Comment:
That's interesting. Apparently IntelliJ automatically adds imports to things that are typos. Adding editing the pull request. 

Please hold.